src/components/AddBook.js

```diff
@@ -0,0 +1,50 @@
+import { useState } from 'react';
+import { v4 as uuidv4 } from 'uuid';
+import { useDispatch } from 'react-redux';
+import { addBook, postBook } from '../redux/books/booksSlice';
+
+export default function AddBook() {
+  const [title, setTitle] = useState('');
+  const [author, setAuthor] = useState('');
+  const dispatch = useDispatch();
+  const handelformSubmit = (e) => {
+    e.preventDefault();
+    if (title && author) {
+      const book = {
+        item_id: uuidv4(),
+        title,
+        author,
+        category: null,
+      };
+      dispatch(postBook(book)).then(() => {
+        dispatch(addBook(book));
+        setTitle('');
+        setAuthor('');
+      });
+    }
+  };
+  return (
+    <div className="form-container">
+      <h2>ADD NEW BOOK</h2>
+      <form onSubmit={handelformSubmit}>
+        <input
+          className="inp-title"
+          type="text"
+          value={title}
+          onChange={(e) => setTitle(e.target.value)}
+          placeholder="Book title"
+        />
+        <input
+          className="inp-author"
+          type="text"
+          value={author}
+          onChange={(e) => setAuthor(e.target.value)}
+          placeholder="Book author"
+        />
+        <button className="add-btn" type="submit">
+          ADD BOOK
+        </button>
```

Comment:
- [ ] Please align the input fields and the button to stretch to the same width as the book cards so your design finally matches the given design. 👍🏽

**Your Design**
<img width="1615" alt="Screenshot 2023-04-08 at 11 27 12 AM" src="https://user-images.githubusercontent.com/50941074/230719455-e7bfcdd3-3b4c-43df-8b11-1e93c2d1d576.png">

**The Original Design**
<img width="1009" alt="Screenshot 2023-04-08 at 11 22 30 AM" src="https://user-images.githubusercontent.com/50941074/230719464-c795fdbf-7a61-4284-a6ed-a926cbdd84ac.png">